lib/linux_admin/security/limits_conf.rb

```diff
@@ -0,0 +1,28 @@
+module LinuxAdmin
+  class Security
+    class LimitsConf
+      CONF_FILE = "/etc/security/limits.conf"
+
+      def self.apply_scap_settings(filename = CONF_FILE)
+        set_value("*", "hard", "core", 0, filename)
+        set_value("*", "hard", "maxlogins", 10, filename)
+      end
+
+      def self.set_value(domain, type, item, value, filename = CONF_FILE)
+        config_text = File.read(filename)
+        new_line = "#{domain} #{type} #{item} #{value}"
+        new_text = config_text.gsub!(/^[^#\n]* #{item} .*/, new_line)
+
+        if new_text
+          File.open(filename, "w") do |file|
+            file.puts(new_text)
+          end
```

Comment:
This pattern simplifies to just `File.write(filename, new_text)`  If you need the extra new_line from the puts at the end you can append it to new_text manually above